succeeding/notifications/README.md

```diff
@@ -44,255 +44,305 @@ In this step, we use Cargo to create a new project and list the dependencies. We
 3. In `lib.rs` list all the functionality we will be using in the following steps.
 
     ```rust
-    use iota_streams::app_channels::api::tangle::{
-        Author,
-        Address,
-        Bytes,
-        Subscriber,
-        Transport,
-        MessageContent,
+    use iota_streams::{
+        app::transport::tangle::{
+            client::Client,
+            PAYLOAD_BYTES,
+        },
+        app_channels::api::tangle::{
+            Author,
+            Address,
+            Bytes,
+            Subscriber,
+            MessageContent,
+        },
     };
     ```
 
-Now we have all the dependencies and functionality listed, we're ready to start coding. Our project should now look like this:
+    Now we have all the dependencies and functionality listed, we're ready to start coding. Our project should now look like this:
+
+    ```
+    src/
+        lib.rs
+    Cargo.toml
+    ```
+
 
-```
-src/
-    lib.rs
-Cargo.toml
-```
+## Step 2. Announce the Channel 
+In this step, we write some functions in the `lib.rs` file for announcing a new Channel which Recipients can listen to in order to receive notifications sent to the channel by the Publisher.
 
+1. In the `lib.rs` file, create a struct called `Publisher` that inherits from `Author`
```

Comment:
Rust has no concept of inheritance and this tutorial is also meant to be an introduction to Rust, so lets not mention it here and rather talk about [encapsulation](https://doc.rust-lang.org/stable/book/ch17-01-what-is-oo.html#encapsulation-that-hides-implementation-details) and exposing interfaces. Maybe something like this is more appropriate:

```suggestion
1. In the `lib.rs` file, create a struct called `Publisher` that encapsulates the `Author` interface. We will use this to build the functions we want our Publisher to expose.
```